demonstrations/tutorial_multiclass_classification.py

```diff
@@ -76,9 +76,10 @@
 # entangling/CNOT gates
 
 def layer(W):
-    qml.Rot(W[0, 0], W[0, 1], W[0, 2], wires=0)
-    qml.Rot(W[1, 0], W[1, 1], W[1, 2], wires=1)
-    qml.CNOT(wires=[0, 1])
+    for i in range(num_qubits):
+        qml.Rot(W[i, 0], W[i, 1], W[i, 2], wires=i)
+    for j in range(int(num_qubits/2)):
+        qml.CNOT(wires=[j*2, j*2+1])
```

Comment:
This approach will pairwise entangle qubits that is qubits 0-1, qubits 2-3, etc will be entangled. Would that be what we'd aim for here?

Reply:
When I ran the code as is after increasing the number of qubits to fit my dataset, I got bad accuracy. After I applied the above modifications, I got 98% accuracy. Could you please suggest and explain the right approach?

Reply:
> When I ran the code as is after increasing the number of qubits to fit my dataset, I got bad accuracy.

Would that mean only having `qml.CNOT(wires=[0, 1])`?

> Could you please suggest and explain the right approach?

Sure! It's great that the accuracy was already such. Another step forward would be to apply even more `CNOT` operations. In the case of having two qubits, applying a single `CNOT` can create entanglement between qubits 0 and 1.

However, as the number of qubits increases, the ability to create pairwise entanglement between any two qubits of the entire system requires applying a CNOT operation between every qubit. That is, for three qubits, we'll need to apply three `CNOT`s, for four qubits four, etc.

This is demonstrated in the [`StronglyEntanglingLayers` template](https://pennylane.readthedocs.io/en/stable/code/api/pennylane.templates.layers.StronglyEntanglingLayers.html?highlight=strongly#pennylane.templates.layers.StronglyEntanglingLayers). This template consists:
1. Applying arbitrary single-qubit rotations on each qubit
2. Pairwise entangling every qubit

As a matter of fact, this code snippet actually corresponds to using `StronglyEntanglingLayers` with `W` weights! Therefore, having a logic similar as [here](https://github.com/PennyLaneAI/pennylane/blob/master/pennylane/templates/layers/strongly_entangling.py#L100) could work: applying CNOTs in a ring structure between every two qubit.

Reply:
So this should achieve the required strong entanglement:
```suggestion
    for j in range(int(num_qubits/2)):
        qml.CNOT(wires=[j*2, j*2+1])
```

Reply:
See this [comment](https://github.com/PennyLaneAI/qml/pull/196#discussion_r553410032).